Applications/Express.md

```diff
@@ -0,0 +1,76 @@
+## Express
+
+The express project is applying to become an incubating
+[Top Level Project of the Node.js Foundation][].
+
+### About Express, User base, Community and Ecosystem
+
+Express is one of the most popular web server frameworks available for Node.js.
+For many Node.js developers, express is their primary interface with Node.js
+on the server.
+
+Express has consistently been among the top packages installed via npm, with
+nearly 4.5 million downloads in just the past month.
+
+It is important to recognize that Express is not the only web framework in the
+Node.js ecosystem and that by incubating Express, neither the Node.js
+Foundation or Node.js core would be "picking a winner" or endorsing Express
+over any other overlapping or similar project.
+
+### Formation of an Express WG
+
+The express project consists of a single top level core repository
+(strongloop/express) that has dependencies on a large number of smaller
+independent components that are currently spread out across multiple
+independent GitHub repositories. This proposal moves only the
+strongloop/express GitHub repository under the Node.js GitHub organization
+(nodejs/express).
+
+If the proposal for incubating express as a top level project is accepted, a
+call for participation would be put out to existing Node.js GitHub organization
+members and an "Express WG" (nodejs/express team in the github organization)
+with commit rights to the nodejs/express repository would be created. This WG
+would also initially include existing contributors to the strongloop/express
+github repository who are not currently members of the Node.js GitHub
+organization.
```

Comment:
Specifically, I have concerns of adding people just to add people, not that people are not welcome. People have always been welcome to Express.js to contribute & likely of the biggest reasons there are not many additional collaborators on the express repository is because all the repository is is mainly dependency update commits, while the real work occurs in other organizations that have various collaborators.